src/theme/app/themeStyles.js

```diff
@@ -381,7 +381,7 @@ export const darktheme = {
   grey313131ToYellowFFF7D8: "#313131",
   grey323232ToGreenF3FFFD: "#323232",
   grey3C3C3CToGreyF1F1F1: "#3C3C3C",
-  grey404040ToGreen3C494F: "#404040",
+  grey404040ToGreen3C494F: "#ffffff66",
```

Comment:
Thanks, @Shiv-2712. Where else is this color used? Can you confirm that the use of this color in other locations on the site are not negatively affected?

Reply:
Welcome! This specific color is exclusively employed in Learn-Components, specifically in the file Chapter Style (chapter.style.js). No other file utilizes this particular color variable.